libvmi/os/windows/core.c

```diff
@@ -523,6 +528,7 @@ init_from_rekall_profile(vmi_instance_t vmi)
 
     status_t ret = VMI_FAILURE;
     windows_instance_t windows = vmi->os_data;
+    addr_t lowest_kernel_address = 0x0;
     dbprint(VMI_DEBUG_MISC, "**Trying to init from Rekall profile\n");
 
```

Comment:
If the user already defined ntoskrln and ntoskrln_va via the config we should skip this part in the Rekall init process that tries to automatically find them. We can skip all the way ahead to the part with `    // Let's do some sanity checking`